browser/css/jsdialogs.css

```diff
@@ -1027,6 +1027,7 @@ input[type='number']:hover::-webkit-outer-spin-button {
 	vertical-align: top;
 	width: -webkit-fill-available;
 	width: -moz-available;
+	max-width: 392px;
```

Comment:
This will affect all the textareas in the whole app.
Please add new selector more specific for the case: eg.  `#YOURDIALOGID #TEXTAREAID.ui-textarea`
Best: at the end of a file with a comment above where we can find such field, thanks for a patch anyway!